Targets/AppStoreConnectUI/Sources/Authorization/AuthView.swift

```diff
@@ -6,27 +6,49 @@
 //
 
 import SwiftUI
+import UniformTypeIdentifiers
 
-struct AuthView: View {
-    @StateObject var viewModel: AuthViewModel
+public struct AuthView: View {
+    @StateObject private var viewModel: AuthViewModel
+    @State private var isImporting: Bool = false
     
-    var body: some View {
+    public init(viewModel: AuthViewModel) {
+        self._viewModel = StateObject(wrappedValue: viewModel)
+    }
+    
+    public var body: some View {
         VStack(alignment: .center) {
             VStack(alignment: .leading) {
                 AuthInputField(title: "Issuer ID", binding: $viewModel.issuerID)
-                AuthInputField(title: "Private Key ID", binding: $viewModel.privateKeyID)
-                AuthInputField(title: "Private Key", binding: $viewModel.privateKey)
             }
             .padding()
             
+            Button("Auth Private Key File") {
```

Comment:
Let's use strings and keys for them? And use swiftgen from L10N target. 

Let's say it will be a key `authorization.button.private-key`

```suggestion
            Button("a") {
```

Reply:
improved